.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# About email notifications, see <https://git.io/JUo27>.
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  # TODO: fill the cron here
+  # schedule:
+  #   - cron: "30 0 * * 1" # Every Monday at half past midnight UTC skip-pr skip-stable
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    env:
+      RUST_BACKTRACE: 1
+      PROJECT_NAME: miri
+    strategy:
+      matrix:
+        build: [linux64, macos, win32]
+        include:
+          - build: linux64
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+          - build: macos
+            os: macos-latest
+            target: x86_64-apple-darwin
+          - build: win32
+            os: windows-latest
+            target: i686-pc-windows-msvc
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
So, uh, where does this define the actual commands that get run?

Reply:
This uses a predefined template used to checkout the build repository.
https://github.com/actions/checkout

Reply:
Oh I see, so while this happens by default on other CI runners, it needs to be specifically included for GH? And no actual tests are run yet? Makes sense.